mmv1/products/cloudbuild/api.yaml

```diff
@@ -230,6 +237,54 @@ objects:
                 exactly_one_of:
                   - github.0.push.0.branch
                   - github.0.push.0.tag
+      - !ruby/object:Api::Type::NestedObject
+        name: 'pubsubConfig'
+        description: |
+          PubsubConfig describes the configuration of a trigger that creates 
+          a build whenever a Pub/Sub message is published.
+        exactly_one_of:
+          - trigger_template
+          - github
+          - pubsub_config
+          - webhook_config
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'subscription'
+            description: |
```

Comment:
Fields that are output only should have the property
```
output: true
```

Reply:
My bad. I missed It here. Thank you. I have updated it now.